### PR TITLE
Add events

### PR DESCRIPTION
**Pull request by PolinaStamenova**
### Add events

- Add events to the buttons, to be able to run the logic (operate.js and calculate.js).

_Thank you for taking time to review my PR!_ 😄  🌵

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,76 +1,180 @@
 import React from 'react';
 import './Calculator.css';
+import calculate from '../logic/calculator';
```
Comment (vichuge) [inline src/components/Calculator.js:3]: - [ ] Both logic scripts `calculator.js & operate.js` could be imported on this component, it looks like you do it in the other logic script, please, add `operate.js` here. 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (PolinaStamenova) [reply src/components/Calculator.js:3]: image

The operate function is only needed in the calculate function it is unnecessary to use the operate function in the calculator itself as it has no use...

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @PolinaStamenova  🔥 

Your app is up and running, you are using states and props like a pro; gitflow, README file and commit messages are excellent, the previous CR ask for the two imports because it's in our guidelines, but it's common sense that you already imported it in the calculator.js, I'm gonna use my common sense and approve your project, nothing more to add here, good job!👍🏽 

ok ok

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by PolinaStamenova** [comment]
Thank you, @mricanho you are awesome :)

**Pull request merged**